### PR TITLE
Use worker-utils version 1.0.28 and clean up dead code  (PLFM-5087)

### PR DESCRIPTION
**Pull request by zimingd**

### CODE REVIEW

File: lib/stackConfiguration/src/main/resources/stack-configuration.spb.xml [hunk 1]
```diff
@@ -15,53 +15,35 @@
...
-	<bean name="stackConfiguration.asyncQueueName[BROADCAST]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[PROJECT_STATS_V2]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[TABLE_UPDATE_TRANSACTION]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[THREAD_STATS]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[MIGRATION]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[DOI]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[ADD_FILES_TO_DOWNLOAD_LIST]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
+	<bean name="stackConfiguration.queueName[LOCATIONABLE_CONVERT_TYPE]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
```
Comment (zimingd) [inline lib/stackConfiguration/src/main/resources/stack-configuration.spb.xml:18]: investigate and try to remove usage of these beans. workers should know the exact queue they need and directly call stackconfiguration

File: lib/stackConfiguration/src/main/java/org/sagebionetworks/StackConstants.java [hunk 2]
```diff
@@ -25,15 +25,13 @@
...
-	public static final String ASYNC_QUEUE_TEMPLATE = "%1$s-%2$s-%3$s-async-queue";
 
-	public static final String WORKER_QUEUE_TEMPLATE = "%1$s-%2$s-%3$s-worker-queue";
+	public static final String QUEUE_TEMPLATE = "%1$s-%2$s-%3$s";
 
 	/**
 	 * Template used the name of the AWS topic where repository changes messages are published.
 	 */
-	public static final String TOPIC_NAME_TEMPLATE_PREFIX = "%1$s-%2$s-repo-";
+	public static final String TOPIC_NAME_TEMPLATE_PREFIX = "%1$s-%2$s-";
```
Comment (zimingd) [inline lib/stackConfiguration/src/main/java/org/sagebionetworks/StackConstants.java:34]: make this same as QUEUE_TEMPLATE.
rename to QUEUE_AND_TOPIC_NAME_TEMPLATE

File: services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/message/RepositoryMessagePublisherImpl.java [hunk 3]
```diff
@@ -92,13 +74,9 @@ public void setAwsSNSClient(AmazonSNS awsSNSClient) {
 	}
 
 
-	private ConcurrentLinkedQueue<Message> messageQueue = new ConcurrentLinkedQueue<Message>();
-
-	public RepositoryMessagePublisherImpl(final String topicPrefix, final String modificationTopicName) {
-		ValidateArgument.required(modificationTopicName, "modificationTopicName");
+	public RepositoryMessagePublisherImpl(final String topicPrefix) {
```
Comment (zimingd) [inline services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/message/RepositoryMessagePublisherImpl.java:77]: remove topic prefix. call stack configuration

File: services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/message/RepositoryMessagePublisherImplTest.java [hunk 4]
```diff
@@ -24,13 +28,17 @@
...
 	ChangeMessage message;
+	@Mock
 	TransactionalMessenger mockTransactionalMessanger;
+	@Mock
 	AmazonSNS mockAwsSNSClient;
-	
-	RepositoryMessagePublisherImpl messagePublisher;
+
+	@InjectMocks
+	RepositoryMessagePublisherImpl messagePublisher = new RepositoryMessagePublisherImpl("prefix");
```
Comment (zimingd) [inline services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/message/RepositoryMessagePublisherImplTest.java:41]: move to before setup method. constructor arg is gone.

**Review by john-hill — APPROVED** [review]

**Pull request merged**